### ISSUE TITLE
Rapport

### ISSUE THREAD

**Issue by arturmatulaniec** [freedrikp/EDA095-Project#9]
Tänkte att vi kanske kunde hålla varandra uppdaterade på rapporten genom en issue. Jag ska bara renskriva min andra rapport sen tänkte jag sätta igång ganska så omgående med denna och fortsätta! :)

**Comment by arturmatulaniec** [comment]
Så, har skrivit en hel del på rapporten! Kommer lägga den lite på is nu, känner mig så trött på den! Ta er en titt, lämna en kommentar alternativt lägg in ändringen själv och så har vi nått bra att lämna in på onsdag :) har kikat på andra rapporter också och måste säga att vi har en bra och utförlig rapport, krävs bra mkt mindre för att bli godkänd.. 

Korrekturläsning och sånt lämnar jag till lite senare, så man har en röd tråd genom hela. Har lagt till lite fotnoter, så vi slipper ha en referenslista! 

Aja, kör hårt grabbar och jag håller kväll för tillfället!

**Comment by freedrikp** [comment]
Gött gött, blev jäkligt överraskad när jag gick in och tittade. Helt färdigt nästan. Jag har skrivit lite mer om protokollet och en hel del på diskussionen. Om synkroniseringen och minnsemöget. Känns rätt färdigt nu.

**Comment by arturmatulaniec** [comment]
Yes, det gör det! Kikade lite snabbt på texten och verkar vara väldigt bra. Det som nu känns mest relevant för rapporten är att få den någorlunda bra, rent språkmässigt. Annars känns det som om vi e Done and Done, har uppdaterat hemsidan med lite filer att ta hem, fast väntar med det till sista stund att ta ned den koden vi faktiskt lämnar in!

**Comment by freedrikp** [comment]
Gott bra jobbat, jag fixade precis minnesskiten så nu lär servern inte krascha, vår linkedlist växte typ alldeles för fort så vi hade typ 600 bilder i den och då dog den.

**Comment by Nisseman3000** [comment]
Goodie, Fredrik 😊 User Guide är skriven, jag vet inte riktigt vad mer det finns att göra i rapporten?

**Comment by freedrikp** [comment]
Fast det streamar inte i realtid så det fuckar ur ändå!

Naj det är nog rätt så bra nu, någon får skicka in idag!!

**Comment by arturmatulaniec** [comment]
Ok ok, bueno :) jag kommer plugga lite senare efter jobb isf, så antingen skickar jag den eller så kanske nån annan hinner före! Annars kommer jag köra på lite infomationsteoriplugg ikväll.

**Comment by arturmatulaniec** [comment]
Jag har nu skickat in rapporten, tror inte vi har något att oroa oss för! Grymt jobbat :) 

Hur ser källkoden ut? Så jag uppdaterar med den senaste på hemsidan, men det e nått vi kan ordna med imorgon innan.
